package.json

```diff
@@ -44,19 +39,14 @@
         "grunt-shell": "^1.3.1",
         "grunt-webpack": "^3.0.2",
         "grunt-zip": "^0.17.1",
-        "jquery": "~3.2.1",
-        "jsoneditor": "~4.1.2",
         "nib": "^1.1.2",
         "pug": "^2.0.0-rc.2",
         "pug-loader": "^2.3.0",
-        "remarkable": "~1.7.1",
-        "sprintf-js": "~1.0.3",
         "style-loader": "^0.13.2",
         "stylus": "^0.54.5",
         "stylus-loader": "^3.0.1",
         "swagger-ui": "~2.2.10",
         "toposort": "^1.0.3",
-        "underscore": "~1.8.3",
```

Comment:
To be clear, it should be in both files now. For consistency with the version convention, this one should now use a `^` version specifier.